### DISCUSSION TITLE
Proposing alternate icons in PRs

### DISCUSSION THREAD

**Discussion by mirrorkeydev** [mirrorkeydev/Appstract#24] [Ideas]
Hey all!

So @The-King-Slayer brought up a good point, that there's no guidelines in the contribution guide on what to do if you'll like to propose multiple icons for one app. Some ideas:

- Add alternatives in the svg (outside the document/artboard), and mention that they exist in the PR (so I know to look for them)
- Add a temporary `alternatives/svg` folder in your PR, where you can put alternatives, and remember to delete it before the PR is merged
- Add alternatives as images in the PR text body. Having proposed icons be images in the PR text body would make reviewing icons a lot quicker and easier for me, but put more work on contributors.

Unfortunately I don't believe I can actually ship Appstract with multiple icons for one app (based on the dashboard I'm using), so that's not a possible option.

@kaugummiheld I'm pinging you because you've been the most active outside contributor to this repo so far (and also the reason it exists haha), and I figure you probably have some good ideas/opinions about this as well :)

**Comment by kaugummiheld** [comment]
Hey you all! :wave: 

Since nobody else seems to have some opinion on this, here's mine:   
My "design process" (not sure you can call my struggles with Inkscape to do what I want) usually ends up with different versions anyway. Some are pretty pure drafts and basically just ideas I had at some point, but others are really more like alternatives.

So therefore I would go with the first option - on the condition of really only having ready alternatives in the final svgs (have to look at myself here).
